### DISCUSSION TITLE
Implementation of getItemOffersBatch , getListingOffersBatch in Products

### DISCUSSION THREAD

**Discussion by kairav646** [saleweaver/python-amazon-sp-api#608] [Feature requests]
Hi,

Amazon added 2 new APIs into (https://developer-docs.amazon.com/sp-api/docs/product-pricing-api-v0-reference#getitemoffersbatch). Can we please add these APIs? Will be very helpful and appreciated.

Thanks!

**Comment by DanielLanger** [comment]
+1, still don't see getListingOffersBatch implemented
